From: Stefan Wahren <stefan.wahren@i2se.com>
Date: Thu, 16 Jun 2016 17:54:06 +0000
Subject: [PATCH 1/1] mmc: add MMC_VERSION_5_1

Signed-off-by: Stefan Wahren <stefan.wahren@i2se.com>
(cherry picked from commit 1a3619cf825fcef15ebe0fac6d65a9af1bc0588c)
Signed-off-by: Gonzalo Ruiz <Gonzalo.Ruiz@digi.com>
---
 drivers/mmc/mmc.c | 3 +++
 include/mmc.h     | 1 +
 2 files changed, 4 insertions(+)

diff --git a/drivers/mmc/mmc.c b/drivers/mmc/mmc.c
index 0bb1588..b7f36ce 100644
--- a/drivers/mmc/mmc.c
+++ b/drivers/mmc/mmc.c
@@ -1216,6 +1216,9 @@ static int mmc_startup(struct mmc *mmc)
 		case 7:
 			mmc->version = MMC_VERSION_5_0;
 			break;
+		case 8:
+			mmc->version = MMC_VERSION_5_1;
+			break;
 		}
 
 		/* The partition data may be non-zero but it is only
diff --git a/include/mmc.h b/include/mmc.h
index 9c9b571..be3333a 100644
--- a/include/mmc.h
+++ b/include/mmc.h
@@ -49,6 +49,7 @@
 #define MMC_VERSION_4_41	MAKE_MMC_VERSION(4, 4, 1)
 #define MMC_VERSION_4_5		MAKE_MMC_VERSION(4, 5, 0)
 #define MMC_VERSION_5_0		MAKE_MMC_VERSION(5, 0, 0)
+#define MMC_VERSION_5_1		MAKE_MMC_VERSION(5, 1, 0)
 
 #define MMC_MODE_HS		(1 << 0)
 #define MMC_MODE_HS_52MHz	(1 << 1)
